functions/create-barrel.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const path = `/${process.argv[2]}`;
```

Comment:
We probably don't need the string template on declaration as we are doing it on line 10.

Reply:
using the path in multiple places so that's why decided to declare it here